packages/web/integrations/axelar/transfer-status-source.ts

```diff
@@ -66,31 +66,32 @@ export class AxelarTransferStatusSource implements ITxStatusSource {
     }
 
     const [data] = transferStatus;
+    const idWithoutSourceChain = data?.id.split("_")[0].toLowerCase();
 
     // insufficient fee
-    if (data.source && data.source.insufficient_fee) {
+    if (data.send && data.send.insufficient_fee) {
       return {
-        id: data.source.id.toLowerCase(),
+        id: idWithoutSourceChain,
         status: "failed",
         reason: "Insufficient fee",
```

Comment:
RE i18n: This is where the failure reason key should be passed. I'm thinking we can create a type with const strings for all the possible failure reasons, then they get mapped to a localization key (and translation) in the view layer. Similar to what we do for `TxStatus`.

Reply:
Nice catch! Added in latest commit